tests/integration/validate_test.go

```diff
@@ -270,3 +271,41 @@ func Test_Validate_Gateway_EE(t *testing.T) {
 		})
 	}
 }
+
+func Test_Validate_PartialLookupTags(t *testing.T) {
+	setup(t)
+	runWhenEnterpriseOrKonnect(t, ">=3.10.0")
+
+	ctx := context.Background()
+
+	tests := []struct {
+		name           string
+		stateFile      string
+		additionalArgs []string
+		errorExpected  bool
+	}{
+		{
+			name:           "validate partials",
+			stateFile:      "testdata/validate/001-partials/partials.yaml",
+			additionalArgs: []string{"--online-entities-list=Partials"},
+		},
+		{
+			name:      "validate default_lookup_tags with partials",
+			stateFile: "testdata/validate/001-partials/partial-lookup-tags.yaml",
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			require.NoError(t, sync(ctx, "testdata/validate/001-partials/partials.yaml"))
```

Comment:
We want to reset gateway for each testcase?

Reply:
not required as there are no other CUD operations.